### DISCUSSION TITLE
VSIX download shows pre-release versions

### DISCUSSION THREAD

**Discussion by rmgouldny** [microsoft/vscode-jupyter#15182] [General]
I run vscode in an offline environment requiring vsix installs for extensions.  

The current release version is 2024.1.1 but the https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter only provides the ability to download 2024.2.2 which is pre-release and will not install.  Historic versions are listed but the high number of pre-release versions do not allow the release version to be displayed.

My workaround was to reverse engineer the website and determine the artifact path. 
https://marketplace.visualstudio.com/_apis/public/gallery/publishers/ms-toolsai/vsextensions/jupyter/2024.1.1/vspackage?targetPlatform=linux-x64

The release version should either be the default download or it should be available on the web page.

**Comment by AndrewUbiety** [comment]
I am to in this exact same situation.   Why are only pre release versions shown that do not work for the stable version of Visual Studio Code ? This is very annoying.  Thank you
